scripts/us_hud/income/process.py

```diff
@@ -33,17 +14,14 @@
 
 
 def get_url(year):
-    '''Return xls url for year.
-
-  Args:
-    year: Input year.
-
-  Returns:
-    xls url for given year.
-  '''
+    '''Return xls url for year.'''
     if year < 2006:
         return ''
     suffix = str(year)[-2:]
+    if year == 2023:
+        return 'Section8-FY23.xlsx'  # Directly reference 2023 file for download
+    elif year == 2024:
```

Comment:
Why are the year and file name hardcoded? How will the script handle future years file?